Jenkinsfile_nightly

```diff
@@ -1,8 +1,7 @@
 #!groovy
 
 properties([
-        // TODO: modify the chron trigger to a time period you want to run them
-        pipelineTriggers([cron('H 02 * * *')]),
+        pipelineTriggers([cron('H 22 * * *')]),
```

Comment:
Does this mean the end to end tests only run overnight? Not when there might be a breaking change?

Reply:
yes, unfortunately at the moment there is no connection between the upload of the CCD definition and the triggering of these tests, so the tests will always run nightly (as a safety net) and then will have to be manually triggered on the env where the changed definition is being tested (e.g. AAT, Demo, etc). (the manual triggering is already in `master`)

Reply:
Ok sounds good